pandas/io/excel/_base.py

```diff
@@ -134,8 +134,24 @@
     as NaN: '""" + fill("', '".join(
     sorted(_NA_VALUES)), 70, subsequent_indent="    ") + """'.
 keep_default_na : bool, default True
-    If na_values are specified and keep_default_na is False the default NaN
-    values are overridden, otherwise they're appended to.
+    Whether or not to include the default NaN values when parsing the data.
```

Comment:
this is handled exactly the same as in pandas/io/parsers.py (e.g. read_csv); don't mind expand this sligthly, but these must be *the* same (also not that this is referenced in io.rst as well).

Reply:
@jreback I'm not able to understand, isn't this the _exact  same_ docstring as that of `keep_default_na` in [`io/parsers.py`](https://github.com/pandas-dev/pandas/blob/b48d1ff87237cb68792590f4377e3be5c73d5ea7/pandas/io/parsers.py#L172-L186)?
If not, should I add the above two lines along with this?